flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/io/checkpointing/AlignedCheckpointsTest.java

```diff
@@ -157,35 +150,6 @@ public void ensureEmpty() throws Exception {
     //  Tests
     // ------------------------------------------------------------------------
 
-    public void testGetChannelsWithUnprocessedBarriers() throws IOException {
-        mockInputGate = new MockInputGate(4, Collections.emptyList());
-        AlignedController alignedController = new AlignedController(mockInputGate);
-        BufferOrEvent barrier0 = createBarrier(1, 0);
-        BufferOrEvent barrier1 = createBarrier(1, 1);
-        BufferOrEvent barrier3 = createBarrier(1, 3);
-        alignedController.barrierAnnouncement(
-                barrier0.getChannelInfo(), (CheckpointBarrier) barrier0.getEvent(), 0);
-        alignedController.barrierReceived(
-                barrier0.getChannelInfo(), (CheckpointBarrier) barrier0.getEvent());
-        alignedController.barrierAnnouncement(
-                barrier1.getChannelInfo(), (CheckpointBarrier) barrier1.getEvent(), 1);
-        alignedController.barrierAnnouncement(
-                barrier3.getChannelInfo(), (CheckpointBarrier) barrier3.getEvent(), 42);
-
-        Collection<InputChannelInfo> blockedChannels = alignedController.getBlockedChannels();
-        Map<InputChannelInfo, Integer> announcedChannels =
-                alignedController.getSequenceNumberInAnnouncedChannels();
-
-        // blockedChannels and announcedChannels should be copies and shouldn't be cleared by the
-        // resumeConsumption
-        alignedController.resumeConsumption();
-
-        assertThat(blockedChannels, contains(barrier0.getChannelInfo()));
-        assertThat(announcedChannels, IsMapContaining.hasEntry(barrier1.getChannelInfo(), 1));
-        assertThat(announcedChannels, IsMapContaining.hasEntry(barrier3.getChannelInfo(), 42));
-        assertThat(announcedChannels.size(), equalTo(2));
-    }
-
```

Comment:
No, it has not been rewritten, as it does not test checkpointing behaviour. It tests properties of `AlignedController`, which does not exist anymore.

Reply:
But in that case it would be nice to rewrite on a higher level, as it was probably there because of some bug, unless there is already a test covering for this?

Reply:
Could you help explaining what does it test then? The way I'm seeing it verifies bookkeeping of sequence numbers.

Tests such as e.g. `testActiveTimeoutAlignmentOnAnnouncement` verify that barriers are proritized on announcements when switching to unaligned checkpoints (data is overtaken)